packages/@react-stately/table/src/useTableState.ts

```diff
@@ -55,10 +63,12 @@ const OPPOSITE_SORT_DIRECTION = {
  */
 export function useTableState<T extends object>(props: TableStateProps<T>): TableState<T> {
   let [isKeyboardNavigationDisabled, setKeyboardNavigationDisabled] = useState(false);
-  let {selectionMode = 'none'} = props;
+  let {selectionMode = 'none', dragAndDropHooks} = props;
+  let isTableDraggable = !!dragAndDropHooks?.useDraggableCollectionState;
```

Comment:
i know it won't change over a components lifetime, but we should add this to the useMemo dep array like you had before on line 75